src/tests/test_pyproxy.py

```diff
@@ -888,6 +888,9 @@ def assert_call(s, val):
     with pytest.raises(selenium.JavascriptException, match=msg):
         selenium.run_js("f.callKwargs(76, {x : 6})")
 
+    assert_call("f.bind({})()", [2, 3])
+    assert_call("f.bind({}).$$ == f.$$", True)
```

Comment:
What about:
```suggestion
    assert_call("f.bind({}) === f", True)
```

Reply:
Directly comparing functions didn't seem to work.

Reply:
Okay. Let's use `===` though.

Reply:
Done